ci/docs/build.sh

```diff
@@ -45,24 +45,24 @@ conda list --show-channel-urls
 
 # Build Doxygen docs
 gpuci_logger "Build Doxygen docs"
-$PROJECT_WORKSPACE/build.sh cppdocs -v
+"$PROJECT_WORKSPACE/build.sh" cppdocs -v
 
 # Build Python docs
 gpuci_logger "Build Sphinx docs"
-cd $PROJECT_WORKSPACE/docs
+cd "$PROJECT_WORKSPACE/docs"
 make html
 
 #Commit to Website
-cd $DOCS_WORKSPACE
+cd "$DOCS_WORKSPACE"
 
 for PROJECT in ${PROJECTS[@]}; do
     if [ ! -d "api/$PROJECT/$BRANCH_VERSION" ]; then
-        mkdir -p api/$PROJECT/$BRANCH_VERSION
+        mkdir -p "api/$PROJECT/$BRANCH_VERSION"
     fi
-    rm -rf $DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*
+    rm -rf "$DOCS_WORKSPACE/api/$PROJECT/$BRANCH_VERSION/*"
 done
 
 
-mv $PROJECT_WORKSPACE/cpp/build/html/* $DOCS_WORKSPACE/api/libcuml/$BRANCH_VERSION
-mv $PROJECT_WORKSPACE/docs/build/html/* $DOCS_WORKSPACE/api/cuml/$BRANCH_VERSION
+mv "$PROJECT_WORKSPACE/cpp/build/html/*" "$DOCS_WORKSPACE/api/libcuml/$BRANCH_VERSION"
+mv "$PROJECT_WORKSPACE/docs/build/html/*" "$DOCS_WORKSPACE/api/cuml/$BRANCH_VERSION"
```

Comment:
```suggestion
mv "$PROJECT_WORKSPACE/cpp/build/html/"* "$DOCS_WORKSPACE/api/libcuml/$BRANCH_VERSION"
mv "$PROJECT_WORKSPACE/docs/build/html/"* "$DOCS_WORKSPACE/api/cuml/$BRANCH_VERSION"
```

the `*` needs to be outside of the quotes to enable shell expansion of file names